Stockwatch/Stockwatch.Businesses/StockDataService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Stockwatch.Model;
+
+namespace Stockwatch.Business
+{
+    public interface IStockDataservice {
+        void AddStock(StockData StockData);
+        void UpdateStock(StockData StockData);
+        void RemoveStock(StockData StockData);
```

Comment:
Same comment as above for both methods

Reply:
Changed to AddStockAlertRange, UpdateStockAlerRange and RemoveAlertRange